Doc/library/sqlite3.rst

```diff
@@ -304,8 +304,9 @@ Module functions
    :param bool check_same_thread:
        If ``True`` (default), only the creating thread may use the connection.
        If ``False``, the connection may be shared across multiple threads;
-       if so, write operations should be serialized by the user to avoid data
-       corruption.
+       if so, write operations may need to be serialized by the user,
+       to avoid data corruption.
+       Please see :attr:`threadsafety` for more information.
```

Comment:
IMO, "Please" changes the meaning of this sentence, from simply informing the user they can consult another resource for more information, to actively begging them to do so. I'm not sure the latter is necessary here, nor particularly consistent with the general tone of these reference docs.

```suggestion
       See :attr:`threadsafety` for more information.
```